math.go

```diff
@@ -8,6 +8,21 @@ type Point struct {
 	X, Y float32
 }
 
+// Abs returns the absolute value of x.
+//
+// Special cases are:
+//	Abs(±Inf) = +Inf
+//	Abs(NaN) = NaN
+func Abs(x float32) float32 {
```

Comment:
We could also just use `luxengine/math`? Otherwise we'd have to implement these methods as we use them, anyway. And there's no downside to using `luxengine/math`?